src/core/bpf-firewall.c

```diff
@@ -375,15 +409,18 @@ static int bpf_firewall_prepare_access_maps(
                 Unit *u,
                 int verdict,
                 int *ret_ipv4_map_fd,
-                int *ret_ipv6_map_fd) {
+                int *ret_ipv6_map_fd,
+                bool *has_any) {
```

Comment:
i would prefer if this was named `ret_has_any` or so, indicating that this is a return parameter